src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Represent an adding or deleting of a tag for a specific person that happened during that session
+ */
+
+public class Tagging {
+	
+	public static final int STATUS_ADDED = 1;
+	public static final int STATUS_DELETED = 2;
```

Comment:
using integers works here. think of how you can use a different data type to make this elegant?